k8s/openebs-monitoring/README.md

```diff
@@ -1,39 +1,82 @@
+├── openebs-monitoring
+│   ├── alertmanager.yaml
+│   ├── configs
+│   │   ├── alertmanager-config.yaml
+│   │   ├── alertmanager-templates.yaml
+│   │   ├── prometheus-alert-rules.yaml
+│   │   ├── prometheus-config.yaml
+│   │   └── prometheus-env.yaml
+│   ├── grafana-operator.yaml
+│   ├── prometheus-operator.yaml
+│   └── README.md
+
+    * alertmanager.yaml : This is for creating deployment and service of alertmanager.
+    * alertmanager-config : This is configuration file for alertmanager, used to load templates and to set alerts in slack. This can be configured to to set alerts at various platforms like e-mail, slack etc.
+    * alertmanager-templates.yaml : This is used to customize notifications send to slack.
+    * prometheus-alert-rules.yaml: Rules defined for alerting is passed  in prometheus-deployment.
+    * prometheus-config.yaml :  This is  configuration file for prometheus, which can be used by prometheus to receive the targets and it displays the collected metrics in the Prometheus expression browser.
+    * prometheus-env: This is to store all the necessary runtime arguments to Prometheus.
+    * grafana-operator.yaml : This file is to create deployment and start service of grafana for various monitoring and analytics.
+    * prometheus-operator.yaml : This is the operator file for prometheus which is used to launch prometheus and it's components on [Kubernetes](https://github.com/kubernetes/kubernetes) cluster.
 
 ## Prerequisits
-* Please go through the steps mentioned in [k8s/vagrant](https://github.com/openebs/openebs/tree/master/k8s/vagrant) subdir .
+* Go through the procedure given in [k8s/vagrant](https://github.com/openebs/openebs/tree/master/k8s/vagrant) subdir .
 
 ## Setup Prometheus
 
 Fetch the latest prometheus-operator.yaml
 ```
-wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/prometheus.yaml
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/configs/prometheus-config.yaml
 wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/prometheus-operator.yaml
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/configs/prometheus-alert-rules.yaml
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/configs/alertmanager-config.yaml
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/configs/alertmanager-templates.yaml
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/configs/prometheus-env.yaml
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/alertmanager.yaml
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/grafana-operator.yaml
+
 ```
-Create configmap and launch prometheus-operator on your kubernetes cluster
+Create configmap and launch prometheus-operator on your Kubernetes cluster
```

Comment:
configmap = configuration map
cluster = cluster.

Reply:
@mahebbar configmap is Kubernetes object.